test/testUtils/mayaUtils.py

```diff
@@ -232,7 +232,12 @@ def previewReleaseVersion():
         return 98
     elif majorVersion == 2020:
         return 110
+    elif majorVersion == 2022:
+        return 122
```

Comment:
I'm curious about this, but maybe I'm misunderstanding.

For the next (hypothetical) preview releases 123, 124, 125, etc., wouldn't they all report their majorVersion as  `2022`, in which case this would return `122` for all of those? I can see how the special cases for 2019 and 2020 are needed since we don't expect there to be any preview releases with those major versions anymore.

Reply:
Yes you are correct, I should not have added these lines. I also jumped the gun on removing the PR123 checks. I realized that PR 123 isn't released yet. I've reverted some of my changes.